internal/storetesting/charm.go

```diff
@@ -272,3 +272,23 @@ func MetaWithResources(m *charm.Meta, resources ...string) *charm.Meta {
 	}
 	return m
 }
+
+// MetaWithCategories returns m with Categories set to categories. If m
+// is nil, new(charm.Meta) will be used instead.
+func MetaWithCategories(m *charm.Meta, categories ...string) *charm.Meta {
```

Comment:
We're not, they seem like they might be useful, but I'll remove them if you prefer.